docs/modules/ROOT/pages/compact-binary-specification.adoc

```diff
@@ -0,0 +1,706 @@
+= Compact Serialization Binary Specification
+:description: This topic describes the binary specification of Compact serialization, showing how serialized data look like in binary format.
```

Comment:
Maybe it is just me, but I feel like the `topic` is not a good word here. Maybe something like `document`, or `page` can be used instead?

Or maybe we can even change the structure of the sentence altogether and say something like

`This specification describes the binary format of the Compact serialization`.

Reply:
It is `This document describes the binary format of the Compact serialization.` now.